packages/shared/src/components/modals/MarketingCtaModal.tsx

```diff
@@ -18,24 +21,45 @@ export const MarketingCtaModal = ({
   onRequestClose,
   ...modalProps
 }: MarketingCtaModalProps): ReactElement => {
+  const { clearMarketingCta } = useBoot();
+  const { trackEvent } = useAnalyticsContext();
   const { tagColor, tagText, title, description, image, ctaUrl, ctaText } =
     marketingCta.flags;
 
   const onModalClose: typeof onRequestClose = (param) => {
```

Comment:
What's the difference between onModalClose and onModalDismiss? Also I see u replaced usage of onModalClose with onModalDismiss, in this case, is onModalClose still being used?

Reply:
onModalDismiss calls onModalClose, it has the mutation call and actually closes the modal.
Didn't want to duplicate that code in onCtaClick and onModalDismiss

Reply:
I refactored them to be one function that takes two arguments.